src/drop-down-menu.jsx

```diff
@@ -65,6 +66,7 @@ let DropDownMenu = React.createClass({
 
   getStyles(){
     let zIndex = 5; // As AppBar
+    let disabled = this.props.disabled;
```

Comment:
I understand what const is, but not the purpose of your line note. Const is not used anywhere else in this file.